cypress/features/regression/experimental/fieldsetClassic.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Experimental Fieldset component
+  I want to change Experimental Fieldset component properties
+
+  Background: Open Experimental Fieldset component page
+    Given I open "Experimental Fieldset" component page classic
+
+  @positive
+  Scenario Outline: Change legend in Fieldset to <legend>
+    When I set legend to "<legend>"
+    Then legend on preview is "<legend>"
+    Examples:
+      | legend                  |
+      | Sample text             |
+      | 1234567890              |
+      | áéíóú¿¡üñ               |
+      | !@#$%^*()_+-=~[];:.,?{} |
+      | ÄÖÜßäöüß                |
+      | <>                      |
+
+  @negative
+  Scenario: Set legend in Fieldset to empty
+    When I set legend to empty
+    Then legend on preview not exists
+
+  @positive
+  Scenario: Verify Fieldset properties
+    # Given I open "Experimental Fieldset" component page
```

Comment:
Why this step is commented out ? If it is needed. add some info

Reply:
It's commented because we don't need to do any actions before we are making `Then` step